### PR TITLE
Added .idea/ to .gitignore

### PR DESCRIPTION
**Pull request by JadeJen**

### CODE REVIEW

**Review by fyy26 — APPROVED** [review]
Yes

**Review by andrewlr09 — APPROVED** [review]
nice

File: .gitignore [hunk 1]
```diff
@@ -37,7 +37,8 @@ ipython_config.py
 .LSOverride
 
 # Icon must end with two \r
-Icon
```
Comment (j6k4m8) [inline .gitignore:40]: Can you take this part of the diff out? This line is deliberately formatted with `\r` which your editor probably didn't like when it saved :)

**Review by j6k4m8 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (fyy26) [reply .gitignore:40]: I wasn't planning to merge this PR - Imma close it

Comment (fyy26) [reply .gitignore:40]: This was just a test PR for github settings : P
But ye u r correct it was prob my editor. I actually even erased my commit and re-did it when I first saw it but it just kept showing up

**Pull request closed without merging**